tsconfig.json

```
@@ -0,0 +1,20 @@
+{
+  "compilerOptions": {
+    "target": "es5",
```

This is what we use in CosmJS:

```suggestion
    "target": "es2018",
```